backend/src/main/java/ai/verta/modeldb/utils/JdbiUtil.java

```diff
@@ -0,0 +1,104 @@
+package ai.verta.modeldb.utils;
+
+import ai.verta.modeldb.common.CommonConstants;
+import ai.verta.modeldb.common.CommonDBUtil;
+import ai.verta.modeldb.common.CommonMessages;
+import ai.verta.modeldb.common.MssqlMigrationUtil;
+import ai.verta.modeldb.config.MDBConfig;
+import io.grpc.health.v1.HealthCheckResponse;
+import java.io.File;
+import java.sql.SQLException;
+import java.util.concurrent.Executor;
+import liquibase.exception.LiquibaseException;
+import liquibase.resource.FileSystemResourceAccessor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.hibernate.exception.JDBCConnectionException;
+
+public class JdbiUtil extends CommonDBUtil {
+  private static final Logger LOGGER = LogManager.getLogger(JdbiUtil.class);
+  private static JdbiUtil jdbiUtil;
+  private static String liquibaseRootFilePath;
+  private static boolean isReady = false;
+  private static MDBConfig config;
+
+  private JdbiUtil() {}
+
+  public static synchronized JdbiUtil getInstance(MDBConfig mdbConfig)
```

Comment:
`static synchronized` is a code smell. If this needs to be made thread-safe, keep a private static field to synchronize on, rather than synchronizing on the class itself.

Reply:
Removed this file from this PR. i will take this in new PR